### DISCUSSION TITLE
Select which Boost libs to install

### DISCUSSION THREAD

**Discussion by s-martin** [MarkusJx/install-boost#35] [Ideas]
Hi,

I didn’t see it so I guess it’s a feature request.

would it be possible to extend the Action so that only selected Boost libs are installed?

while I understand that this could lead to dependency errors, I think it would be helpful, if you need only one or two libs.

Great tool, btw! 🎉🎉🎉

**Comment by MarkusJx** [comment]
Currently, I'm not planning on doing something like that, due to the added complexity.
Also, as you've mentioned, adding features which may lead to errors seems not like a good idea to me.

Do you have any specific use case where you'd just need a few libraries?
Or is it just a performance-related issue? Because that doesn't seem like a problem, since installing boost with caching enabled takes at most 50 seconds.
